docs/docs/guides/04-providers/03-email-http-api.md

```diff
@@ -0,0 +1,139 @@
+---
+id: email-http
+title: Using HTTP Email APIs with Auth.js
+---
+
+## Introduction
+
+We have always provided an Email provider with Auth.js with which you could provide to any SMTP server of your choice to send use "magic link" emails to sign-in with. However, as time went on, we had more and more people as us to make it easy to use HTTP-based Email providers, like AWS SES, Postmark, Sendgrid, etc. This was theoretically always possible, but we didn't make it as straight-forward as possible. In this guide I'm going to explain how to use our Email magic link provider with any of the more modern HTTP-based Email APIs.
+
+For this API I will be using [Sendgrid](https://sendgrid.com), but any Email provider which provides an HTTP API / JS-client library will do the trick.
+
+## Setup 
+
+First, clone and setup a basic Auth.js project like the the one [provided in our example repo](https://github.com/nextauthjs/next-auth-example.git). I won't go into the details of setting up your project for Auth.js's [Email provider](https://next-auth.js.org/providers/email), but you will need to make at least the following modifications to the example repository, or any other project you're adding this support to, including:
+
+  - Install `nodemailer` as a dependency. The email provider also requires the use of a database adapter for Auth.js.
```

Comment:
```suggestion
```

the point of this guide is to not need `nodemailer`

Reply:
I removed the text actually since its a default dependency in the example-repo as well.

Reply:
> It'll complain if you don't install it

Right, because we have the import at the top of the module... :facepalm: Then I would not use `EmailProvider` at all. The whole point of this tutorial is to get rid of `nodemailer`, it feels weird to recommend installing it, and it's also quite big.